string aStr
string bStr
---
string opStr
